payjoin-directory/docker-compose.yml

```
@@ -0,0 +1,55 @@
+version: '3.8'
+
+services:
+  nginx:
+    image: nginx:latest
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ./nginx/logs:/var/log/nginx
+      - ./nginx/conf.d:/etc/nginx/conf.d
+      - ./nginx/certs:/etc/ssl/certs
+      - ./nginx/html:/var/www/html
+    networks:
+      - payjoin-network
+
+  certbot:
+    image: certbot/certbot
+    volumes:
+      - ./nginx/certs:/etc/letsencrypt
+      - ./nginx/html:/var/www/html
+    entrypoint: /bin/sh -c 'trap exit TERM; while :; do certbot renew --webroot -w /var/www/html --deploy-hook "nginx -s reload"; sleep 12h & wait $${!}; done;'
+    depends_on:
+      - nginx
+    networks:
+      - payjoin-network
+
+  payjoin-directory:
+    image: dangould/payjoin-directory:0.0.1-rc0
```

Should this be:
```suggestion
    image: dangould/payjoin-directory:0.0.1
```
?

---

It should definitely be, once that tag is released